### PR TITLE
Uniform pilot-agent and pilot-discover 's code style

### PR DESCRIPTION
**Pull request by vince-0202**
**Please provide a description of this PR:**

Uniform pilot-agent and pilot-discover 's code style.

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [X ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ X] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pilot/cmd/pilot-agent/app/request.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ var (
 )
 
 func init() {
-	rootCmd.AddCommand(requestCmd)
+
```
Comment (GregHanson) [inline pilot/cmd/pilot-agent/app/request.go:79]: remove extra new line to pass lint/gen error

File: pilot/cmd/pilot-agent/app/wait.go [hunk 2]
```diff
@@ -82,5 +82,4 @@ func init() {
 	waitCmd.PersistentFlags().IntVar(&periodMillis, "periodMillis", 500, "number of milliseconds to wait between attempts")
 	waitCmd.PersistentFlags().StringVar(&url, "url", "http://localhost:15021/healthz/ready", "URL to use in requests")
```
Comment (GregHanson) [inline pilot/cmd/pilot-agent/app/wait.go:84]: remove empty line at end go func() to make lint/gen happy

**Review by GregHanson — COMMENTED** [review]
thanks for cleaning up your earlier PR, just missing a few small nits to get all the tests green

**Comment by GregHanson** [comment]
@vince-0202 I'm not sure which editor you are currently using, but if you are using VS code, here are some of the Go settings I use to auto format to keep the linters happy
```
    "go.formatFlags": [
        "[\"-ed\", \"-local\", \"istio.io\"]"
    ],
    "go.formatTool": "goimports",
```

**Comment by vince-0202** [comment]
@GregHanson Thanks，It‘s useful！

**Comment by vince-0202** [comment]
@GregHanson okay，I will pay special attention to this in the future.

**Review by GregHanson — APPROVED** [review]

**Pull request merged**